skipper.go

```diff
@@ -503,11 +503,12 @@ type Options struct {
 	OIDCSecretsFile string
 
 	// API Monitoring feature is active (feature toggle)
-	ApiUsageMonitoringEnable                       bool
-	ApiUsageMonitoringRealmKeys                    string
-	ApiUsageMonitoringClientKeys                   string
+	ApiUsageMonitoringEnable                bool
+	ApiUsageMonitoringRealmKeys             string
+	ApiUsageMonitoringClientKeys            string
+	ApiUsageMonitoringRealmsTrackingPattern string
+	// Deprecated: ApiUsageMonitoringDefaultClientTrackingPattern
```

Comment:
Can you please use `// *DEPRECATED* ` as we already with in other comments? It's easier to have always the same comment.